js/common/lib/trace.ts

```diff
@@ -51,3 +51,23 @@ export const TRACE_FUNC_END = (extraMsg?: string) => {
   }
   TRACE_FUNC('END', extraMsg);
 };
+
+/**
+ * @ignore
+ */
+export const TRACE_EVENT_BEGIN = (extraMsg?: string) => {
```

Comment:
if you always specify the message, you can use `extraMsg: string` instead of `extraMsg?: string`.

Reply:
Yes, message will be always specified, fixed this issue and updated the PR, please help review.